pkg/controller/cronjob_scheduler/cronjob_scheduler_controller.go

```diff
@@ -0,0 +1,267 @@
+package cronjob_scheduler
+
+import (
+	"context"
+	"fmt"
+	"log"
+	"os"
+	"strings"
+	"time"
+
+	cnoclient "github.com/openshift/cluster-network-operator/pkg/client"
+	"github.com/openshift/cluster-network-operator/pkg/controller/statusmanager"
+	"github.com/openshift/cluster-network-operator/pkg/names"
+	"github.com/openshift/cluster-network-operator/pkg/render"
+	"github.com/pkg/errors"
+	appsv1 "k8s.io/api/apps/v1"
+	corev1 "k8s.io/api/core/v1"
+	apierrors "k8s.io/apimachinery/pkg/api/errors"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/apis/meta/v1/unstructured"
+	"k8s.io/apimachinery/pkg/runtime"
+	"k8s.io/apimachinery/pkg/types"
+	utilruntime "k8s.io/apimachinery/pkg/util/runtime"
+	"k8s.io/apimachinery/pkg/util/wait"
+	v1coreinformers "k8s.io/client-go/informers/core/v1"
+	"k8s.io/client-go/tools/cache"
+	"k8s.io/klog/v2"
+	crclient "sigs.k8s.io/controller-runtime/pkg/client"
+	"sigs.k8s.io/controller-runtime/pkg/controller"
+	"sigs.k8s.io/controller-runtime/pkg/handler"
+	"sigs.k8s.io/controller-runtime/pkg/manager"
+	"sigs.k8s.io/controller-runtime/pkg/predicate"
+	"sigs.k8s.io/controller-runtime/pkg/reconcile"
+	"sigs.k8s.io/controller-runtime/pkg/source"
+)
+
+const (
+	manifestDir              = "bindata/cronjob-scheduler/daemonset"
+	cronConfigmapDsName      = "cron-scheduler-configmap-ds"
+	cronConfigmapManifestDir = "bindata/cronjob-scheduler/config/configmap.yaml"
```

Comment:
Hrmmm, ok I see now that you're referencing this here. Now I'm not sure it the files from that dir should be in the bindata/network/multus because those get rendered, and you want to use it here. Now I think I get it.

I'm now not sure where files like this should go. Can you find another examples of something else in the CNO that references files like this and see where they're stored?

Reply:
I based it off the allowlist, so my other example would be `bindata/allowlist/config/configmap.yaml`.